Resources/Prototypes/Entities/Mobs/NPCs/shadows.yml

```diff
@@ -0,0 +1,71 @@
+- type: entity
+  id: BaseShadowMob
+  abstract: true
+  parent: BaseMob
+  components:
+  - type: Flashable
+  - type: DestroyedByFlashing
+    lifeCount: 2
+  - type: Sprite
+    color: "#793a80dd"
+  - type: PointLight
+    color: "#793a80dd"
+    radius: 1.1
+  - type: MovementSpeedModifier
+    baseWalkSpeed: 2
+    baseSprintSpeed: 2
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.35
+        density: 120
+        mask:
+        - Opaque
+        layer:
+        - None
+  - type: StandingState
+  - type: Tag
+    tags:
+    - DoorBumpOpener
+  - type: NpcFactionMember
+    factions:
+    - SimpleNeutral
+  - type: HTN
+    rootTask:
+      task: IdleCompound
+  - type: SlowOnDamage
+    speedModifierThresholds:
+      60: 0.7
+      80: 0.5
+      
+- type: entity
+  name: shadow cat
+  parent: BaseShadowMob
```

Comment:
should also inherit a cat mob prototype so it keeps the npc stuff like eating mice and would mean less duplication

Reply:
this turned out to be a bad idea, since the cat also inherits blood, damage taking, and other things that aren't needed.

Reply:
fair